Providers/Scripts/2.4x-2.5x/Scripts/nxOMSPerfCounter.py

```diff
@@ -259,18 +274,49 @@ def UpdateOMSAgentConf(WorkspaceID, HeartbeatIntervalSeconds, PerfCounterObject)
     return True
 
 
-def rm_unicode(obj):
-    if isinstance(obj, dict):
-        d = {}
-        for k, v in obj.iteritems():
-            d[rm_unicode(k)] = rm_unicode(v)
-        return d
-    elif isinstance(obj, list):
-        return [rm_unicode(i) for i in obj]
-    elif isinstance(obj, unicode):
-        return obj.encode('ascii', 'ignore')
+def CheckForOMIMappingPathInConf():
+    """
+    Return true if the omi_mapping_path has been specified in all perf
+    sections in conf_path
+    """
+    conf_path_txt = get_conf_path_text()
+    sources = search_for_perf_sections(conf_path_txt)
+    for source in sources:
+        try:
+            if 'omi_mapping_path' not in source[4]:
+                return False
+        except:
+            return False
+    return True
+
+
+def get_conf_path_text():
+    """
+    Get current text in the conf_path file
+    """
+    txt = ''
```

Comment:
Same here, please use python string.empty instead of '' throughout the entire source file.

Reply:
Same comment